tests/Wallabag/CoreBundle/CoreKernelTestCase.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace tests\Wallabag\CoreBundle;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+abstract class CoreKernelTestCase extends KernelTestCase
+{
+    protected $fetchingErrorMessage;
+
+    public function setUp()
+    {
+        self::bootKernel();
+        $container = self::$kernel->getContainer();
+        $this->fetchingErrorMessage = $container->getParameter('wallabag_core.fetching_error_message');
```

Comment:
It was to avoid duplication. But OK, we are in tests, so we can duplicate the string 👍